compiler/rustc_interface/src/tests.rs

```diff
@@ -542,6 +542,7 @@ fn test_codegen_options_tracking_hash() {
     tracked!(debug_assertions, Some(true));
     tracked!(debuginfo, 0xdeadbeef);
     tracked!(embed_bitcode, false);
+    tracked!(export_executable_symbols, false);
```

Comment:
```suggestion
    tracked!(export_executable_symbols, true);
```
the value needs to be different from the default value here.

Reply:
thanks you, I'd fix it soon